README.md

```diff
@@ -420,7 +420,7 @@ $ curl -X PUT "localhost:1338/latest/api/token" -H "X-aws-ec2-metadata-token-ttl
       <title>400 - Bad Request</title>
    </head>
    <body>
-      <h1>400 - Bad Request</h1>
+      <h1.4.1 - Bad Request</h1>
```

Comment:
haha oops

Reply:
twice in a week 🙄 

thanks for catching it